docs/example-react-router.mdx

```diff
@@ -3,7 +3,159 @@ id: example-react-router
 title: React Router
 ---
 
-This example demonstrates React Router v6. For previous versions see below.
+This example demonstrates React Router v6.4 and above. For previous versions see below.
+
+```jsx
+// App.jsx
+import {
+  createBrowserRouter,
+  Form,
+  Link,
+  Outlet,
+  redirect,
+  RouterProvider,
+  useLoaderData,
+  useLocation,
+} from 'react-router-dom';
+
+// Method to introduce an artificial delay
+export function sleep(n = 500) {
```

Comment:
Could you clean this example up please?
`sleep` doesn't really add any value here, it's just introduces clutter.

Reply:
Actually this is a modified form of the example in react router [examples source code](https://github.com/remix-run/react-router/blob/main/examples/data-router/src/app.tsx)  I wanted this to be close to the original doc and hence left it.